src/PIL/ImageShow.py

```diff
@@ -218,6 +218,20 @@ def get_command_ex(self, file, title=None, **options):
         return command, executable
 
 
+class IPythonViewer(Viewer):
+    def show_image(self, image, **options):
+        display(image)
+        return 1
+
+
+try:
+    from IPython.display import display
```

Comment:
```suggestion
    from IPython.display import display as _display
```

Would it be reasonable to add a prefix to avoid confusion when using code completion tools? `ImageShow.show` and `ImageShow.display` are very similar and could be confusing to someone using it for the first time.

Reply:
Okay, sure. I went with `ipython_display`, to be even clearer.